sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py

```diff
@@ -237,6 +246,99 @@ def get_custom_model(self, model_id, **kwargs):
         response = self._client.get_custom_model(model_id=model_id, include_keys=True, error_map=error_map, **kwargs)
         return CustomFormModel._from_generated(response)
 
+    @distributed_trace
+    def authorize_copy_target(self, resource_id, resource_region, **kwargs):
```

Comment:
`authorize_copy_target` doesn't use the parameters `resource_id` and `resource_regino`, right?

Reply:
Correct; I'm adding them to the method because the target should pass these values, not the source resource. I put them in the dictionary that is returned and then passed into `begin_copy_model`